packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -77,6 +77,7 @@ const GlobalHeader = ({ className }) => {
 
   const hideLogoText = useMedia({ maxWidth: '655px' });
   const useSearchIcon = useMedia({ maxWidth: '585px' });
+  const domain = window.location.hostname;
```

Comment:
We wont hav access to `window` at build time. When a site that uses this code builds, it will be in a purely node environment (not in a browser) so there's no concept of a window.

That said, we should be able to query for it using GraphQL. If you add `siteUrl` to the query above (in the `siteMetada` block), that should have the information you're looking for.